tests/python_keep_opening_on_previous_line/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,243 @@
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ],
+    1
+)))
+
+foo(bar(
+    baz([
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ]),
+    1
+))
+
+foo(
+    bar(baz([
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ])),
+    1
+)
+
+long_func_1_aaaaaaaaaaaaa(
+    long_func_2_aaaaaaaaaaaaaaaaa(long_func_3_aaaaaaaaaaaaaa(1))
+)
+
+
+def foo():
+    return [
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ]
+
+
+def foo():
+    yield [
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa,
+        long_item_1_aaaaaaaaaaaa
+    ]
+
+`;
+
+exports[`keep_opening_on_previous_line.py 2`] = `
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = 1 / my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_3_aaaaaaaaaaaaaaaaaaaaaa = my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+a = [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+a = b = [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa])))
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa], 1)))
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]), 1))
+
+foo(bar(baz([long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa])), 1)
+
+long_func_1_aaaaaaaaaaaaa(long_func_2_aaaaaaaaaaaaaaaaa(long_func_3_aaaaaaaaaaaaaa(1)))
+
+def foo():
+    return [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+
+
+def foo():
+    yield [long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa, long_item_1_aaaaaaaaaaaa]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_aaaaaaaaaaaaaaaaaaaaaa = \\
+    1 / my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
+
+my_long_variable_1_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_variable_2_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_variable_3_aaaaaaaaaaaaaaaaaaaaaa = \\
+    my_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
You should be able to get the result that you want without conditional groups using carefully crafted groups, indents and ifBreak.